java/jakarta/servlet/jsp/resources/jspxml.xsd

```diff
@@ -25,7 +25,7 @@
 <!ENTITY SetProp    "(&Identifier;|\*)">
 <!ENTITY RelativeURL  "[^:#/\?]*(:{0,0}|[#/\?].*)">
 <!ENTITY Length     "[0-9]*&#x25;?">
-<!ENTITY AsciiName    "[A-Za-z0-9_-]*">
+<!ENTITY AsciiName    "[\w-]*">
```

Comment:
The Tomcat project is unable to change these schema. We have to use the ones provided by the specification. Please direct this part of the PR to the Jakarta Schema project:
https://github.com/eclipse-ee4j/jakartaee-schemas